odoo/tests/common.py

```diff
@@ -348,6 +348,22 @@ def __init__(self, methodName='runTest'):
         self.addTypeEqualityFunc(etree._Element, self.assertTreesEqual)
         self.addTypeEqualityFunc(html.HtmlElement, self.assertTreesEqual)
 
+    def run(self, result):
+        ODOO_TEST_FAILURE_RETRIES = int(os.environ.get('ODOO_TEST_FAILURE_RETRIES', 1))
```

Comment:
```suggestion
        ODOO_TEST_FAILURE_RETRIES = int(os.environ.get('ODOO_TEST_FAILURE_RETRIES', 0)) + 1
```
seems more logical, if somebody sets `ODOO_TEST_FAILURE_RETRIES=0` they want the tests to not be retried (run up to once) and `ODOO_TEST_FAILURE_RETRIES=1` they want the tests to be retried once (run up to twice).

And the local should then be renamed to something more like `tests_run_count`.